articles/container-registry/container-registry-firewall-access-rules.md

```diff
@@ -19,7 +19,7 @@ To pull or push images or other artifacts to an Azure container registry, a clie
 
 * **Registry REST API endpoint** - Authentication and registry management operations are handled through the registry's public REST API endpoint. This endpoint is the login server name of the registry. Example: `myregistry.azurecr.io`
 
-* **Registry REST API endpoint for certificates** - Azure container registry uses a wildcard SSL certificate for all subdomains. When connecting to the Azure container registry using SSL, the client must be able to download the certificate for the TLS handshake. In such cases, `azurecr.io` must also be accessible.
+  * **Registry REST API endpoint for certificates** - Azure container registry uses a wildcard SSL certificate for all subdomains. When connecting to the Azure container registry using SSL, the client must be able to download the certificate for the TLS handshake. In such cases, `azurecr.io` must also be accessible.
```

Comment:
There is a clear description. We don't require a sub bullet.

Reply:
https://github.com/MicrosoftDocs/azure-docs/pull/114553/files/8f5ff4aa51f4ce1e9eb606e01565688bbb36a4ba#diff-b683422a71586f98661d32441e009ce93732cfc7b76aaadcbf43d299886a4b51L16

To be clear, I modify this into sub-bullet because it is under the title "About registry endpoints". The word "for certificates" clearly indicates that this is the subscription to certificate, not directly for endpoints.

This is why I change it to sub-bullet since this is not direct description to "endpoints". This may the only little barrier for strong-logic readers to understand.  But I was trying to make it in a more clear way to reduce time cost to understand it.